app/views/devise/passwords/edit.html.haml

```diff
@@ -4,20 +4,17 @@
   %span
 .entry-content.wpcf7
   = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
-    - if resource.errors.any?
-      .wpcf7-response-output
-        - resource.errors.full_messages.each do |msg|
-          = msg
+
+    != model_errors_helper(resource)
+    %br/
```

Comment:
Yes, please enter a bug.  We could add some margin to the top-most div containing to the error text in order to separate the error section from the text above it.